app/css/cohortbuilder.css

```diff
@@ -203,4 +203,8 @@ a:hover {
 
 #concept-verbose {
     text-indent: '20px';
+}
+
+#error-message {
+    color: #ff0000;
 }
```

Comment:
Awesome PR @kingisaac95 @paulupendo and millicent, please can we add an empty line to this file.